effects/zio/src/test/scala/sttp/client3/impl/zio/ZioTestBase.scala

```diff
@@ -28,5 +29,17 @@ trait ZioTestBase {
   }
 
   def timeoutToNone[T](t: Task[T], timeoutMillis: Int): Task[Option[T]] =
-    t.timeout(timeoutMillis.milliseconds).provideLayer(Clock.live)
+    t.timeout(timeoutMillis.milliseconds).provideLayer(ZLayer.succeed[Clock](ClockLive)(Tag[Clock], Tracer.newTrace))
```

Comment:
You can remove `provideLayer` entirely (or use `ZIO.withClock` to force using live clock)